package.json

```diff
@@ -11,5 +11,8 @@
     "lint": "cd frontend && npm run lint",
     "postinstall": "npm run build && npm run deploy",
     "start": "cd frontend && npm start"
+  },
+  "dependencies": {
+    "react-select": "^3.0.8"
```

Comment:
All dependencies go in the `frontend/package.json` file, not this one. You can remove these with `git checkout package.json && rm package-lock.json`

Reply:
This appears to still be here. You can just remove the changes (basically this whole dependencies block).